modules/pagerduty/client.go

```diff
@@ -7,14 +7,17 @@ import (
 )
 
 // GetOnCalls returns a list of people currently on call
-func GetOnCalls(apiKey string) ([]pagerduty.OnCall, error) {
+func GetOnCalls(apiKey string, scheduleIDs []string) ([]pagerduty.OnCall, error) {
```

Comment:
So, it probably makes sense to support the rest of the query options at some point
Do we just take a ListOnCallOptions as a param? Or just worry about that later?